docs/conf.py

```diff
@@ -117,10 +117,7 @@
 # Theme options are theme-specific and customize the look and feel of a
 # theme further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = {
-    "show_relbars": True,
-    "extra_nav_links": {"GitHub": "https://github.com/SFDO-Tooling/CumulusCI"},
-}
+# html_theme_options = {}
```

Comment:
This negates a couple of errors that occur while building the docs with the new theme.